superset/translations/utils.py

```diff
@@ -16,15 +16,15 @@
 # under the License.
 import json
 import os
-from typing import Any, Dict
+from typing import Any, Dict, Optional
 
 # Global caching for JSON language packs
 ALL_LANGUAGE_PACKS: Dict[str, Dict[Any, Any]] = {"en": {}}
 
 DIR = os.path.dirname(os.path.abspath(__file__))
 
 
-def get_language_pack(locale):
+def get_language_pack(locale: str) -> Optional[Dict[Any, Any]]:
```

Comment:
I believe we could make the key `str` to improve differentiation slightly without complicating things too much (strictly speaking it seems the structure is `Optional[Dict[str, Union[str, Dict[str, Union[Dict[str, str], List[str]]]]]]` which quite frankly looks awful and probably won't help detect any errors)